app/src/main/java/edu/uc/hickmadc/customnotifications/ui/main/GroupsFragment.kt

```diff
@@ -30,8 +30,10 @@ class GroupsFragment : Fragment() {
         val groups: ArrayList<Group> = viewModel.getAllGroups()
         val adapter =
             GroupsAdapter(groups)
-        binding.groupList.adapter = adapter
-        binding.fabAdd.setOnClickListener { viewModel.exampleClick() }
+        with(binding){
```

Comment:
Implemented with() so there is no need to use same dot operators multiple times

Reply:
Good.